From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Cheng Zhao <zcbenz@gmail.com>
Date: Tue, 9 Apr 2019 14:57:02 -0700
Subject: disable_network_services_by_default.patch

Disable NetworkService by default.

This allows us to enable NetworkService with command line flags, which makes it
easier to work on NetworkService migration.

We should remove this patch after all Electron's code has been migrated to the
NetworkService.

diff --git a/services/network/public/cpp/features.cc b/services/network/public/cpp/features.cc
index fc98e3f39a754642fcad5507deae2e4ba9672a05..7bdd77dd20b62ee93ede03d2a9159a4f8f218e0a 100644
--- a/services/network/public/cpp/features.cc
+++ b/services/network/public/cpp/features.cc
@@ -27,7 +27,7 @@ const base::Feature kNetworkErrorLogging{"NetworkErrorLogging",
                                          base::FEATURE_ENABLED_BY_DEFAULT};
 // Enables the network service.
 const base::Feature kNetworkService{kNetworkServiceFeatureName,
-                                    base::FEATURE_ENABLED_BY_DEFAULT};
+                                    base::FEATURE_DISABLED_BY_DEFAULT};
 
 // Out of Blink CORS
 const base::Feature kOutOfBlinkCors{"OutOfBlinkCors",
